portal/src/LoadingModal.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import ShowLoading from "./ShowLoading";
+
+import styles from "./LoadingModal.module.scss";
+
+interface LoadingModalProps {
+  loading: boolean;
+}
+
+const LoadingModal: React.FC<LoadingModalProps> = function LoadingModal(
+  props: LoadingModalProps
+) {
+  if (!props.loading) {
+    return null;
+  }
+
+  return (
+    <div className={styles.root}>
+      <ShowLoading />
+    </div>
+  );
+};
+
+export default LoadingModal;
```

Comment:
Can we simply show a [Spinner](https://developer.microsoft.com/en-us/fluentui#/controls/web/spinner) next to the save button with a label telling the user that we are saving? When we are saving, set pointer events to none to prevent user interaction.